tasks/create_client.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+- name: Set client facts
+  set_fact:
+    openvpn_client_client_name: "{{ openvpn_client.client_name | default(openvpn_client_default_client_name) }}"
+    openvpn_client_server_name: "{{ openvpn_client.server_name | default(openvpn_client_default_server_name) }}"
+    openvpn_client_up_commands: "{{ openvpn_client.up_commands | default(openvpn_client_default_up_commands) }}"
+    openvpn_client_extra_config: "{{ openvpn_client.extra_config | default(openvpn_client_default_extra_config) }}"
+
+- name: Set client facts
+  set_fact:
+    openvpn_client_config_identifier: "{{ openvpn_client_client_name }}-{{ openvpn_client_server_name }}"
```

Comment:
Couldn't these three `set_fact` tasks be merged?

Reply:
No

Reply:
![image](https://media1.tenor.com/images/cb89abffb06b3fabb69dfab551e3a0ac/tenor.gif?itemid=5783187)

Reply:
Yes